Download all .fastq files in a project/Download all .fastq files in a project.ipynb

```diff
@@ -0,0 +1,6347 @@
+{
```

Comment:
<p>This <code>{"query"</code> portion should be formatted as a code block and run through a json formatter for readability</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCellAtlas/data-consumer-vignettes/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>